sigle/src/hooks/appData.ts

```diff
@@ -44,6 +44,7 @@ export const useUserFollow = () => {
       body: { stacksAddress: address, createdAt: now },
     });
     await queryClient.invalidateQueries('get-users-followers');
+    await queryClient.invalidateQueries('get-users-following');
```

Comment:
both lines could be merged in just one invalidateQueries call to make it run faster.

Reply:
Also another question, as we are using `['get-users-following', userAdress]` as the key, what exactly is this one doing? removing the cache for all the users ?

Reply:
I think this was added before the fix/update adding the key - should this be updated to include the user address?

Reply:
I am not sure about this, probably the docs have the answer for this one :)

Reply:
after investigating the docs, it seems that this will invalidate any matching queries so yeah would essentially remove the cache for all the users.

Another thing I noticed is that now we are getting following/followers amount data from the userInfo object, the number is no longer updating if you follow/unfollow a user.

Reply:
In that case I think the best way to fix this is to update the user info optimistically the same way we do above with get-user-following, wdyt?

Reply:
yep that makes sense 👍

regarding merging them into one invalidateQueries call, is the syntax meant to be something like this: 
`await queryClient.invalidateQueries(['get-users-followers', 'get-users-following', 'get-user-by-address'])` ? 

tried this and it is not working - also checked the docs and can't seem to find info about invalidating multiple queries with a different key. Do you know of another way or should I just call them separately?

Reply:
Looks like this is only possible with the v4 of the module so let's keep it separate for now 👍